wordpress-mobile/gutenberg-mobile
# Move the post title selection state to the store and update PostTitle

This is a testing PR for https://github.com/WordPress/gutenberg/pull/16704

### Testing Instructions:

Boot the app locally and try the instructions in https://github.com/wordpress-mobile/gutenberg-mobile/pull/1219 to make sure there is no regression when adding a new block
